modules/sdk/project-templates/project-templates/build.gradle

```diff
@@ -174,6 +176,26 @@ uploadProjectTemplateArchives {
 	}
 }
 
+projectTemplateJarVersions {
+	File projectTemplateVersionsFile = new File(buildDir, "project-template-jar-versions.properties")
+
+	Properties properties = new Properties()
```

Comment:
The code on `L182` can be moved after `outputs` to follow the `as used` principle.